src/pages/cats.js

```diff
@@ -343,6 +360,20 @@ const Cats = ({ pageContext }) => {
                 our users, and a great way to make an impact without breaking
                 the bank.
               </p>
+              {isReferral ? (
+                <p
+                  data-test-id={'referral-text'}
+                  style={{
+                    width: '80%',
+                    textAlign: 'justify',
+                    fontWeight: 'bold',
+                  }}
+                >
+                  Since you were referred to join Tab for Cats, we'll give an
+                  additional donation to shelter cats on behalf of you and your
+                  referral when you sign up!
```

Comment:
Maybe: "Your friend sent you a gift / By signing up with this link, you'll immediately help shelter cats get adopted by giving them 5 treats for positive-reinforcement training."